# request
Point location

---
# response
bool success
string message